2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,25 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    this.volume = 100
   },
   pour: function () {
     // calling this function should increase your bottle volume by 10 units;
+    if (this.volume<=90){this.volume +=10}
   },
   drink: function () {
     // calling this function should decrease your bottle volume by 10 units;
+    if (this.volume>=10){this.volume-=10}
   },
   isFull: function () {
     // this function should return true if your bottle is full;
+  if(this.volume == 100){return true}
```

Comment:
when you have an have an if statement such as:
```
if (condition) { boolean}
```
you can simplify as:
``` 
condition
```
in this case:

```
return this.volume == 100
```

Reply:
thank you changed it to shorter version